### PR TITLE
Setup start config ⚛️

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Setting up base configuration
* Code Climate and Travis CI config
* Published starter files on npm https://www.npmjs.com/package/jason-react-form

### ⚠️ Most of the files was generated from create-react-app

**PRINTS** :framed_picture:
⛔️

### CODE REVIEW

File: src/components/StartText.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class StartText extends React.Component {
```
Comment (nathpaiva) [inline src/components/StartText.jsx]: porque não:

`import React, { Component } from 'react';`?

File: src/index.dev.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './App.jsx';
```
Comment (nathpaiva) [inline src/index.dev.js]: porque jsx?

File: src/components/StartText.jsx (hunk 1 above)
Comment (iondrimba) [reply src/components/StartText.jsx]: Because 1: Linting configuration 2: You write less code 😃

Comment (felipeorlando) [reply src/components/StartText.jsx]: @nathpaiva I agree with it... we just need to change the eslint config to not warning in the `React` assign statement, that is not used in the code.

**Comment by iondrimba** [comment]
Yes, we are using create-react-app. We can use it and if we find something we wont use, we can remove from the configuration.

File: src/index.dev.js (hunk 2 above)
Comment (felipeorlando) [reply src/index.dev.js]: I add this config on eslint

**Pull request merged**